core/modules/layout/layout.module

```diff
@@ -2768,3 +2768,17 @@ function _layout_clean_custom_css($classes = array()) {
 function layout_field_delete_instance($instance) {
   cache('layout_path')->flush();
 }
+
+/**
+ * Render a block
+ * @param Block $block
+ * The Block entity to render
+ */
+function layout_render_block($block) {
+  $output = '';
+  $render_array = $block->getRenderArray();
+  if ($render_array) {
+    $output = backdrop_render($render_array);
+  }
+  return $output;
```

Comment:
If `getRenderArray()` (aka `view()`) always returns an array then this can be simplified:

```suggestion
  return backdrop_render($block->view());
```

`backdrop_render()` returns early if the array is empty.